app/registries/foreman/access_permissions.rb

```diff
@@ -286,6 +286,7 @@
     map.permission :power_hosts,   {:hosts          => [:power],
                                     :"api/v2/hosts" => [:power, :power_status] }
     map.permission :console_hosts, {:hosts => [:console] }
+    map.permission :host_provisioning_template, {:provisioning_templates => [:preview] }
```

Comment:
Should I add this new permission to a role? Like SiteManager,  If I would name it like "view_host_provisioning_templates" (just to differenciate it from provisioning_template -> view_provsisioning_template) then it would be automatically be added to viewer_permissions? 
@MariaAga

Reply:
found a different solution. Hope you like it :)